### PR TITLE
UOE-10484: Fix- Inject error tracker in Adm for AppLoinMax

### PR DESCRIPTION
**Pull request by AvinashKapre**

### CODE REVIEW

File: modules/pubmatic/openwrap/tracker/banner_test.go [hunk 1]
```diff
@@ -22,6 +22,23 @@ func Test_injectBannerTracker(t *testing.T) {
...
+			args: args{
+				rctx: models.RequestCtx{
+					Platform: models.PLATFORM_APP,
+					Endpoint: models.EndpointAppLovinMax,
+				},
+				tracker: models.OWTracker{
+					TrackerURL: `Tracking URL`,
+				},
+				bid: openrtb2.Bid{
+					AdM: `sample_creative`,
```
Comment (pm-nilesh-chate) [inline modules/pubmatic/openwrap/tracker/banner_test.go:36]: use valid banner adm

File: modules/pubmatic/openwrap/tracker/inject_test.go [hunk 2]
```diff
@@ -481,6 +481,137 @@ func TestInjectTrackers(t *testing.T) {
 			},
 			wantErr: false,
 		},
+		{
+			name: "adformat_is_banner_and_AppLovinMax_request",
```
Comment (pm-nilesh-chate) [inline modules/pubmatic/openwrap/tracker/inject_test.go:485]: use valid adm

File: modules/pubmatic/openwrap/tracker/native_test.go [hunk 3]
```diff
@@ -10,16 +10,28 @@ import (
...
 	}{
+		{
+			name: "AppLovinMax_endpoint_request",
+			args: args{
+				native:   &openrtb2.Native{},
+				adm:      `creative`,
+				tracker:  models.OWTracker{},
+				endpoint: models.EndpointAppLovinMax,
+			},
+			want:    `creative`,
```
Comment (pm-nilesh-chate) [inline modules/pubmatic/openwrap/tracker/native_test.go:32]: use valid adm

File: modules/pubmatic/openwrap/tracker/inject.go [hunk 4]
```diff
@@ -30,10 +30,10 @@ func InjectTrackers(rctx models.RequestCtx, bidResponse *openrtb2.BidResponse) (
 				bidResponse.SeatBid[i].Bid[j].AdM = injectBannerTracker(rctx, tracker, bid, seatBid.Seat, pixels)
 			case models.Video:
 				trackers := []models.OWTracker{tracker}
-				bidResponse.SeatBid[i].Bid[j].AdM, err = injectVideoCreativeTrackers(bid, trackers)
+				bidResponse.SeatBid[i].Bid[j].AdM, err = injectVideoCreativeTrackers(rctx, bid, trackers)
 			case models.Native:
 				if impBidCtx, ok := rctx.ImpBidCtx[bid.ImpID]; ok {
-					bidResponse.SeatBid[i].Bid[j].AdM, err = injectNativeCreativeTrackers(impBidCtx.Native, bid.AdM, tracker)
+					bidResponse.SeatBid[i].Bid[j].AdM, err = injectNativeCreativeTrackers(impBidCtx.Native, bid.AdM, tracker, rctx.Endpoint)
```
[the remark this answers is not in the document]
Comment (AvinashKapre) [reply modules/pubmatic/openwrap/tracker/inject.go:36]: Here in case of Native we have diffrent handling so kept different

**Review by pm-saurabh-narkhede — APPROVED** [review]

**Pull request merged**